client/state/billing-transactions/schema.js

```diff
@@ -6,47 +6,12 @@ export const billingTransactionsSchema = {
 			type: 'array',
 			items: {
 				type: 'object',
-				properties: {
```

Comment:
We use schemas to protect against data shape changes. An example use case would be that the API changes a prop name, we have some old stored data with the old prop name, and the client code assumes that the new prop name exists. With a schema we can detect the change and throw out the old values instead of running into a js error.

 So the approach here would be to update the schema or relax it, or to not persist at all.